ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/deploy_container/scripts/dockerCPU.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/bash
+
+# Usage - Script takes one argument, the number of cpus for the docker container
+# ie bash dockerCPU.sh 4
+# Argument cannot be 0 or greater than the number of CPUs on the machine
+# If /var/log/docker_cpu does not exist, run: echo 0 > /var/log/docker_cpu
+
+set -eu
+
+cpu_limit=$1
+start_cpu=$(tail -n 1 /var/log/docker_cpu)
+max_cpus=$(lscpu | grep "CPU(s):" | head -n 1 | awk '{split($0,a," "); print a[2]}')
+cpus_to_use=""
+
+if [ $cpu_limit -gt $max_cpus ] || [ $cpu_limit == "0" ]
```

Comment:
We should probably check these values to ensure they are actual numbers before doing the calculation e.g. if an error condition when reading/parsing the output.